content/tutorials/manage-the-rippled-server/troubleshooting/server-doesnt-sync.md

```diff
@@ -0,0 +1,110 @@
+# rippled Server Doesn't Sync
+
+This page explains possible reasons [a `rippled` server](the-rippled-server.html) may start successfully, but get stuck in a ["connected" state](rippled-server-states.html) without ever fully connecting to the network. (If the server crashes during or shortly after startup, see [Server Won't Start](server-wont-start.html) instead.)
+
+These instructions assume you have [installed `rippled`](install-rippled.html) on a supported platform.
+
+
+## Normal Syncing Behavior
+
+Syncing with the network normally takes about 5 to 15 minutes. During that time, the server does several things:
+
+- Loads a recommended validator list (typically from `vl.ripple.com`) to determine which validators it trusts.
+- [Discovers peer servers](peer-protocol.html#peer-discovery) and connects to them.
+- Downloads the [header](ledger-header.html) and full [state information](ledgers.html#tree-format) of the latest ledger from its peers, and uses that to build its internal database of ledger data.
+- Listens to its trusted validators to find which ledger hashes have been recently validated.
+- Collects newly-broadcast transactions and attempts to apply them to its in-progress ledger.
+
+If the server is unable to keep up with the network while doing these tasks, the server does not sync to the network.
+
+
+## First Step: Restart
+
+Many syncing issues can be resolved by simply restarting the server. Even if it failed to sync for whatever reason, it may succeed on the second try.
+
+If the [server_info method][] shows a [`server_state`](rippled-server-states.html) other than `proposing` or `full` and a `server_state_duration_us` of more than `900000000` (15 minutes in microseconds), then you should shut down the `rippled` service, wait a few seconds, and start it again. Optionally, restart the entire machine.
+
+If the problem persists, check the other possibilities listed on this page. If none of them seem to apply, [open an issue in the `rippled` repository](https://github.com/ripple/rippled/issues) and add the "Syncing issue" label.
+
+
+## Usual Causes of Syncing Issues
+
+The most common cause of syncing issues is not meeting the [system requirements](system-requirements.html). The three most common shortfalls are:
+
+- **Slow disks.** You need a consistently fast solid state disk (SSD). Cloud providers like AWS usually don't guarantee disk performance, which may be impacted by other users of shared hardware.
```

Comment:
I have limited knowledge of AWS, but it seems you can get an instance with a default SSD with a certain level of guaranteed performance with AWS, and that many high-intensity applications are run on AWS. Is there a way to quantify this?